src/openrct2/title/TitleScreen.cpp

```diff
@@ -132,6 +133,10 @@ void TitleScreen::Load()
     CreateWindows();
     TitleInitialise();
     OpenRCT2::Audio::PlayTitleMusic();
+    if (gConfigSound.audio_focus)
```

Comment:
No. This bool is changed when the checkbox "Disable audio on focus loss" is checked or unchecked.
Same logic is used in file `src/openrct2-ui/UiContext.cpp` on line 369.
There I have added additional checks for mouse focus and window taking focus.
A moment ago I have tested it once more and I my opinion checking for mouse focus only should be sufficient.

Reply:
@ZehMatt you were right. I have been mistaken. I've tested game once more and added event polling to `TitleScreen::Load()` function. I am hoping, that this (and other changes made) won't be an issue. Till now I haven't found any better place or solution, but as always comments are appreciated, so that I might upgrade the solution.